### DISCUSSION TITLE
DSD 256 buffering problem

### DISCUSSION THREAD

**Discussion by marcogas74** [MusicPlayerDaemon/MPD#2061] [Q&A]
Chi, I use mpd 0.23.15 on volumio 3 . I have a raspberry pi 3 with Innomaker DAC.
I have problem with DSD 256 files that sound bad, the sound is cut to pieces, snappering. I know the problem is initial buffering, I set audio_buffer_size to 12Mb but it does not change anything.
Previously, using mpd 0.20.23, setting audio_buffer_size to 12Mb and buffer_before_play to 20% the same files sound great!!
Since mpd 0.21 buffer_before_play was set to 1 second, independent of audio format. 
I tried increasing to 3 seconds editing source (Thread.cxx in /src/player) and compiling mpd 023.15, the problem persists. Should I increase more?
Is there a way to increase the initial buffering, having the same result I had with mpd 0.20.23?
Thanks, Marco

**Comment by MaxKellermann** [comment]
But what is the real problem here? Start with https://www.musicpd.org/help/
